docs/index.md

```diff
@@ -2,57 +2,69 @@
 
 ## Overview
 
-Open Traffic Generator (OTG) APIs and Data Models are APIs to test Layer 2 & 3 routers and switches. They include support for testing the forwarding plane as well as testing the control plane.  They are:
+Open Traffic Generator (OTG) APIs and data models are northbound API specifications for modern Traffic Generators and Protocol Emulators designed to test Layer 2-7 network devices. They include support for testing  forwarding as well as control planes. OTG APIs are:
 
 * Open
-* Vendor Neutral
-* Intent Based
+* Vendor-neutral
+* Intent-based
 * Declarative
 
-The specification is available on GitHub [here](https://github.com/open-traffic-generator/models/blob/master/artifacts/openapi.yaml) (viewer friendly version at reDocly [here](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/open-traffic-generator/models/master/artifacts/openapi.yaml)).  Specifications support both REST or gRPC interfaces.
+The formal [model specification](https://github.com/open-traffic-generator/models/blob/master/artifacts/openapi.yaml) can be found on GitHub under [Open Traffic Generator](https://github.com/open-traffic-generator) organization. To explore the model, a viewer friendly [**ReDoc rendering**](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/open-traffic-generator/models/master/artifacts/openapi.yaml) is available as well. The OTG APIs support both REST and gRPC interfaces.
 
 ## Features
 
-OTG is an actively developing specification, with contributions coming from real [use cases](/examples/#use-cases). On high level, the model allows to express the following building blocks of a traffic generator configuration:
- 
-* *Test Ports* with:
-	- LLDP
-	- LAG
-	- LACP
-* *Traffic Flows* associated with:
-	- Test Ports
-	- Emulated Devices
-* *Emulated Devices* with:
-	- BGP
-	- IS-IS
+OTG is an actively developing specification, with contributions from real [use cases](/examples/#use-cases). The model allows to express the following building blocks of a traffic generator configuration:
```

Comment:
1. "actively developed" instead of "actively developing"? 
2. Rephrase last sentence as "The model allows to express configuration of the following building blocks of a traffic generator"?

Reply:
1. done
2. The model defines the following components of a traffic generator configuration: